content/en/docs/Configuration/amplify-agents.md

```diff
@@ -12,43 +12,40 @@ The Amplify agents are software applications that run on your host. The agents a
+Traceability Agents collect usage, metrics, and dataplane traffic details and sends them to the Amplify platform. In the platform, API consumers and API providers can view the performance and behavior of the assets discovered in the dataplane.
 
 ## Amplify agents for Axway Open Banking
 
-Axway Open Banking solution embbeds the Discovery and tracability agent fo Axway API Management. It gather information about the Open Banking APIs to the Amplify platform.
+The Axway Open Banking solution embeds the Discovery and traceability agent for Axway API Management. It gathers information about the Open Banking APIs to the Amplify platform.
 
 ### Amplify configuration
 
-#### Service Account Creation
+This section includes details for Amplify configuration such as creating a service account, creating an environment, finding the organization ID and team details.
 
-We must first create a service account in [Amplify platform](https://platform.axway.com).
+#### Service Account Creation
 
-In the drop down menu from your user, select Organization and then "Service Accounts".
+You must first create a service account in the [Amplify platform](https://platform.axway.com).
 
+1. In the drop down menu from your user click Organization - **Service Accounts**.
 ![Service Account in Amplify](/Images/agents/service-account-img1.PNG)
-
-Make sure the right organization is selected.
-
-Create a new service account.
-
+2. Check the Service Accounts drop down to make sure you have the correct organization selected.
+3. Create a new service account.
 ![Service Account Creation in Amplify](/Images/agents/service-account-img3.PNG)
-
-Select a Name.
-
-For the method pick "Client Certificate" and for credentials select "Platform-generated key pair". If you have your own certificate, you can also select the first option, "Provide public key".
-
-You can add one or more organisations and teams, but the default options would work.
-
-A pop-up window will apear with the private key. Download it and keep it, it will be used in the values.yaml file.
+4. Select a Name.
+5. Complete the following fields.
+    * **Method**: select **Client Certificate**.
+    * **Credentials**: Select **Platform-generated key pair** or **Provide public key** if you have your own certificate.
+6. You can add one or more organizations and teams, but the default options would work.
+7. Click **Save**.
+8. A pop-up window will appear with the private key. Download it and save a copy as it will be used in the `values.yaml` file.
 
 ![Service Account Private key window in Amplify](/Images/agents/service-account-privatekey.PNG)
 
-The service account client id, name, public and private key  will be used in the values.yaml file in the following parameters:
+The service account client id, name, public, and private key will be used in the `values.yaml` file in the following parameters:
```

Comment:
I always have trouble getting the voice of the sentence right. Should this be in the active voice or does it matter..

Reply:
As discussed, I'll update voice in another PR.